src/libraries/System.Runtime.Numerics/src/System/Numerics/BigNumber.cs

```diff
@@ -286,6 +287,8 @@ internal static class BigNumber
                                                            | NumberStyles.AllowThousands | NumberStyles.AllowExponent
                                                            | NumberStyles.AllowCurrencySymbol | NumberStyles.AllowHexSpecifier);
 
+        private static readonly uint[] s_uint32PowersOfTen = { 1, 10, 100, 1000, 10000, 100000, 1000000, 10000000, 100000000, 1000000000 };
```

Comment:
With this, we'll now have at least two "powersOfTen" tables in the framework, each private, and each in a different assembly.

@stephentoub, any ideas on how we could (or if we should) share this? It's not a lot, but it still is 40 bytes + IL for each.

In both cases, it's to support a `BigInteger` type (which we have a couple specializations of because this one doesn't meet the requirements in other places)

Reply:
> any ideas on how we could (or if we should) share this? It's not a lot, but it still is 40 bytes

Not really.